packages/local_storage/lib/src/local_storage.dart

```diff
@@ -26,51 +21,55 @@ class LocalStorage {
       version: 1,
       onCreate: (db, version) {
         db.execute('''
-          CREATE TABLE $_tasksTableName(
-            $_tasksIdColumnName INTEGER PRIMARY KEY,
-            $_tasksContentColumnName TEXT NOT NULL,
-            $_tasksStatusColumnName INTEGER NOT NULL
+          CREATE TABLE tasks(
+            id INTEGER PRIMARY KEY,
+            conctent TEXT NOT NULL,
+            status INTEGER NOT NULL
           )
           ''');
       },
     );
     return database;
   }
 
-  Future<void> addTask(String content) async {
+  /// Add new document to the table database
+  Future<void> addDocument({
+    required String collection,
+    required Map<String, dynamic> document,
```

Comment:
as we use SQL in this code and we have 'CREATE TABLE' command, it may be easier to understand this code if we stick to table convention.